lead_tagger/__init__.py

```
@@ -0,0 +1,2 @@
+from .base import BaseTagger
```

Ruff (0.8.2)

`.base.BaseTagger` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)